crates/bevy_reflect/src/lib.rs

```diff
@@ -1745,7 +1745,7 @@ bevy_reflect::tests::should_reflect_debug::Test {
 
     #[test]
     fn multiple_reflect_lists() {
-        #[derive(Hash, PartialEq, Reflect)]
+        #[derive(Hash, PartialEq, Reflect, FromReflect)]
```

Comment:
This doesn't need `FromReflect` since it's a test (and sometimes deriving `FromReflect` could break the robustness of the test).